backend/src/authentication/authentication.service.ts

```diff
@@ -22,25 +22,21 @@ export class AuthenticationService {
         if (!user) { throw new BadRequestException(content.UserNotFound); }
         return await this.validateUser({
             providedPassword: loginData.password, hashedPassword: user.password,
-        }, {
-            id: user.id, identity: user.email,
-        });
+        }, user.id);
     }
 
     async loginJunior(loginData: LoginJuniorDto): Promise<JWTToken> {
         const user = await this.juniorService.getJunior(loginData.phoneNumber);
         if (!user) { throw new BadRequestException(content.UserNotFound); }
         return await this.validateUser({
             providedPassword: loginData.pin, hashedPassword: user.pin,
-        }, {
-            id: user.id, identity: user.pin,
-        });
+        }, user.id);
     }
 
-    async validateUser(attempt: { providedPassword: string, hashedPassword: string }, user: { id: string, identity: string }): Promise<JWTToken> {
+    async validateUser(attempt: { providedPassword: string, hashedPassword: string }, user: string): Promise<JWTToken> {
```

Comment:
Perhaps `userId` as you passed user.id on line 33?

Reply:
These should get resolved in the next merge which is reliant on a major re-structure.